<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="CORPLPW100314" failures="3" tests="4" name="Karthika Tests" time="74.416" errors="0" timestamp="2021-04-13T13:56:33 IST">
  <testcase classname="com.athmatest.qa.testcases.SampleTesting" name="myFirstTest" time="0.044">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.athmatest.qa.testcases.SampleTesting.myFirstTest(SampleTesting.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- myFirstTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTesting" name="mySecondTest" time="9.538">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.athmatest.qa.testcases.SampleTesting.mySecondTest(SampleTesting.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- mySecondTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTesting" name="myThirdTest" time="8.155">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.athmatest.qa.testcases.SampleTesting.myThirdTest(SampleTesting.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- myThirdTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTesting" name="myFourthTest" time="5.881"/>
</testsuite> <!-- Karthika Tests -->
